domain/src/main/java/life/qbic/apps/datamanager/notifications/MessageParameters.java

```diff
@@ -16,18 +16,18 @@ public final class MessageParameters {
 
   public final String messageId;
 
-  public final String occuredOn;
+  public final String occurredOn;
 
-  private MessageParameters(String messageType, String messageId, String occuredOn) {
+  private MessageParameters(String messageType, String messageId, String occurredOn) {
```

Comment:
well spotted!

Reply:
The yellow underline from IntelliJ helped a lot here :)